src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -94,7 +103,7 @@ public function viewAction(Request $request, $cartId)
     /**
      * Gets requested cart information
      *
-     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller')) || is_granted('create', 'AdminOrders')")
```

Comment:
I understand it's the same thing as with customer But again this seems dangerous

Reply:
@matks ping for concerns of security. I don't want to have all this on my shoulders alone :smile:

Reply:
So the matter is the following:

This is an AJAX route that can be used from either a customer page or an order page.
Then comes the question: if I use this customer ajax route on a order page,
- does the access rights to order page grant me access to this route too ?
- do I require both access rights to order page and customer page to be able to use this page ?
- in this last case, what happens if I have been granted only order page access rights, and not customer ? ideally the feature would be disabled

This is a behavior question that needs to be discussed with product managers. I think what @zuk3975 did is right (allowing Orders access rights grants the user the required rights to fully use all features on the order page) but this is a global access rights structure strategy that needs to be discussed.

As this can be changed in the future without introducing BC break, I think it can be discussed after QA validation 👍